client/state/comments/utils.js

```diff
@@ -0,0 +1,58 @@
+/***
+ * Creates a comment target id, a concatenation of siteId and postId basically
+ * @param {Number} siteId site identification
+ * @param {Number} postId post identification
+ * @returns {String} comment target id
+ */
+export function createCommentTargetId( siteId, postId ) {
+	return `${ siteId }-${ postId }`;
+}
+
+/***
+ * Creates a request id, a concatenation of siteId, postId, and query params basically
+ * @param {Number} siteId site identification
+ * @param {Number} postId post identification
+ * @param {Object} query post identification
+ * @returns {String} request id
+ */
+export function createRequestId( siteId, postId, query ) {
+	return `${siteId}-${postId}-${ JSON.stringify( query ) }`;
```

Comment:
I don't think we can stringify the query safely and use it as a key. key order can change which will change the key, even though the objects are equivalent.